### PR TITLE
ISSD-3944 [Testray] Fix error in Pinned Filters

### PR DESCRIPTION
**Pull request by rafaelfs98**

### CODE REVIEW

**Comment by rafaelfs98** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

2 files changed, 78 insertions(+), 14 deletions(-)

**Comment by rafaelfs98** [comment]
ci:test:sf

File: workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/context/ListViewContext.tsx [hunk 1]
```diff
@@ -305,18 +307,51 @@ const ListViewContextProvider: React.FC<
...
 	useEffect(() => {
-		if (!filter && filterSchemaStorage && filterPinnedStorage) {
+		const filters =
+			filterPinnedStorage?.filter &&
+			Object.keys(filterPinnedStorage?.filter).map((key) => {
+				if (Array.isArray(filterPinnedStorage?.filter[key])) {
+					return {
+						name: key,
+						value: (filterPinnedStorage?.filter as any)[key].map(
+							(options: Option) => options.value || options
```
Comment (JoMaAlves) [inline workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/context/ListViewContext.tsx:317]: This part seems a bit strange, due to the any. But if it is the only way that made it work, for me is okay.

**Review by JoMaAlves — APPROVED** [review]
LGTM ​🦖​🌋​ 
Thank you @rafaelfs98

File: workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/context/ListViewContext.tsx (hunk 1 above)
Comment (JoMaAlves) [reply workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/context/ListViewContext.tsx:317]: @rafaelfs98

**Comment by rafaelfs98** [comment]
@joseabelenda

**Comment by joseabelenda** [comment]
resent:https://github.com/brianchandotcom/liferay-portal/pull/148868

**Pull request closed without merging**